src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue

```diff
@@ -6,20 +6,155 @@
           <h1 class="mb-3">
             Let’s work on your justification for this exception to fair opportunity
           </h1>
-          <p>
-            Intro paragraph
+          <p v-if="selectedException === 'YES_FAR_16_505_B_2_I_B'">
+            Briefly describe how this action requires use of the “unique or highly
+            specialized capabilities” exception. Discuss the events, history, and
```

Comment:
In all 3 cases, the vast majority of the text is exactly the same. Please use getters to fill in the blanks
```Briefly describe how this action requires use of the "_____" exception. Discuss the events, history, and circumstances that led to the current situation requiring use of procedures for other than fair opportunity competition. Your rationale should _____. ```